third-party/py/pex/pex/link.py

```diff
@@ -73,7 +73,9 @@ def __init__(self, url):
     """
     purl = urlparse.urlparse(url)
     if purl.scheme == '':
-      purl = urlparse.urlparse(self._normalize(url))
+      purl = urlparse.urlparse(self._normalize(url), allow_fragments=False)
+    elif purl.scheme == 'file':
+      purl = urlparse.urlparse(url, allow_fragments=False)
```

Comment:
This also adds support for file, right?  Care to update the message and `README.facebook` to be more clear?